sdk/monitor/azure-monitor-opentelemetry-exporter/src/samples/java/com/azure/monitor/opentelemetry/exporter/ReadmeSamples.java

```diff
@@ -44,8 +44,7 @@ public void setupExporter() {
         // Create Azure Monitor exporter and initialize OpenTelemetry SDK
```

Comment:
please update the comment, it seems to be out of sync with the code

```suggestion
        // Configure OpenTelemetry to export data to Azure Monitor
```